src/config.ts

```diff
@@ -28,4 +19,18 @@ export const config = {
         required: true,
         secret: process.env.SECRET_KEY || 'bLue5tream@2018', // Don't use static value in production! remove from source control!
     },
+    cors: {
+        allowedOrigins: process.env.ALLOWED_ORIGINS ? process.env.ALLOWED_ORIGINS.split(',') : ['http://localhost:4200'],
+    },
+    s3: {
+        region: process.env.S3_REGION || '',
+        bucket: process.env.S3_BUCKET || '',
+        accessKeyId: process.env.S3_ACCESS_KEY_ID || '',
+        secretAccessKey: process.env.S3_SECRET_ACCESS_KEY || '',
+        signatureVersion: process.env.S3_VERSION || 'v4',
+        endpoint: process.env.S3_ENDPOINT || '',
+    },
+    streamer: {
+        maxChunkSize: 1024 * 1024 // 1Mb
```

Comment:
Add env var to max chunk size

```suggestion
        maxChunkSize: process.env.MAX_CHUNK_SIZE_IN_MB * 1024 || 1024 * 1024 // 1Mb
```